src/mam4xx/calcsize.hpp

```diff
@@ -610,8 +610,9 @@ void aitken_accum_exchange(
     const Real &adj_tscale_inv, const Real &dt, const Prognostics &prognostics,
```

Comment:
Not important, but there's little point in passing scalars by const reference instead of by value. A pointer address is just as expensive to copy as a scalar.